src/EA.Weee.Web/Areas/AatfReturn/Views/CheckYourReturn/Index.cshtml

```diff
@@ -95,53 +95,55 @@
 
             if (aatf.SchemeData.Count > 0)
             {
-                <details class="govuk-details collapsible-font-size" open="open">
-                    <summary id="summary-collapsible" class="govuk-details__summary" aria-controls="details-content" aria-expanded="false" onclick="return CollapsibleLinkHefSummary();">
-                        <span id="unset-padding" class="govuk-details__summary-text" onclick="return CollapsibleLinkHefSummary();">
-                            Breakdown by PCS
-                        </span>
-                    </summary>
-                    <div class="govuk-!-padding-bottom-4"></div>
-                    <div class="collapsible-govuk-grid">
-                        <div class="govuk-grid-row govuk-body collapsible-font-size">
-                            <div class="govuk-grid-column-one-half">
-                                <div class="govuk-grid-column-two-thirds">
-                                    <strong>PCS Name</strong>
-                                </div>
+    <details class="govuk-details collapsible-font-size" open="open">
+        <summary id="summary-collapsible" class="govuk-details__summary" aria-controls="details-content" aria-expanded="false" onclick="return CollapsibleLinkHefSummary();">
+            <span id="unset-padding" class="govuk-details__summary-text" onclick="return CollapsibleLinkHefSummary();">
+                Breakdown by PCS
+            </span>
+        </summary>
+        <div class="govuk-details__text">
+            <div class="govuk-!-padding-bottom-4"></div>
+            <div class="collapsible-govuk-grid">
+                <div class="govuk-grid-row govuk-body collapsible-font-size">
+                    <div class="govuk-grid-column-one-half">
+                        <div class="govuk-grid-column-two-thirds">
+                            <strong>PCS Name</strong>
```

Comment:
Getting a merge conflict here, looks like the title changes haven't been made here e.g. PCS Name should be PCS name, Approval number etc

Reply:
Fixed :)